mapping_cpu.impala

```diff
@@ -83,48 +83,56 @@ struct CacheSizes {
 }
 
 fn tiled_loop(xl: i32, xu: i32, yl: i32, yu: i32, mask: Mask, img: Img, body: fn(i32, i32) -> ()) -> () {
+    let debug_tiling = false;
     fn get_tile_dims(mask: Mask, img: Img) -> (i32, i32) {
         // TODO make this dimension independent, currently only 2D is supported
         // TODO get cache information from machine/compiler/somewhere else
 
         let caches = CacheSizes {sizes: [32*1024, 256*1024, 20*1024*1024], count: 3}; // in bytes
         let element_size = sizeof[i32](); // TODO make this dynamic based on used type. templates?
         
-        //print_string("img: (");
-        //print_int(img.width);
-        //print_string(", ");
-        //print_int(img.height);
-        //print_string(" : ");
-        //print_int(img.stride);
-        //print_string(")\n");
-        //
-        //print_string("mask: (");
-        //print_int(mask.size_x);
-        //print_string(", ");
-        //print_int(mask.size_y);
-        //print_string(")\n");
+        if debug_tiling @{        
```

Comment:
No @ is required here - the frontend will already discard the conditional code if ```debug_tiling``` is false.